src/workspacer.FSharpConfig/Extensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace workspacer.FSharpConfig
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+
+    public static class Extensions
+    {
+        public static Assembly LoadFile(this AppDomain appDomain, string filename) {
+            FileStream fs = new FileStream(filename, FileMode.Open);
+            byte[] buffer = new byte[(int) fs.Length];
+            fs.Read(buffer, 0, buffer.Length);
+            fs.Close();
```

Comment:
Any reason not to just use `var bytes = File.ReadAllBytes(filename)` ?

Reply:
Reading a stream like this is known to be unreliable, and you should always return the *actual* amount of bytes read returned from the `FileStream.Read()`-invocation.